modules/k8s-nmstate-creating-interfaces.adoc

```diff
@@ -0,0 +1,250 @@
+// This is included in the following assemblies:
+//
+// k8s-nmstate-configuration.adoc
+
+[id="creating-interfaces_{context}"]
+
+= Creating interfaces
+
+The `kubernetes-nmstate` operator requires the `nmstate.io` API version and the kind of policy, which is `NodeNetworkConfigurationPolicy`. Additionally, each policy has a name as defined by `metadata.name` and a desired state as defined by `spec.desiredState`. The policy's desired state contains a declarative specification of the node's network configuration.
+
+The `spec.desiredState.interfaces` section begins the `nmstate` policy. Creating the policy is identical to creating a policy with `nmstate`. Refer to link:nmstate.io[NMState] documentation for additional details on creating `nmstate` policies.
+
+.Procedure
+
+. Create a policy manifest.
++
+[source,bash]
+----
+ ---
+ apiVersion: nmstate.io/v1beta1
+ kind: NodeNetworkConfigurationPolicy
+ metadata:
+   name: bond0-eth1-eth2
+ spec:
+   desiredState:
+     interfaces:
+     - name: bond0
+       type: bond
+       state: up
+       ipv4:
+         dhcp: true
+         enabled: true
+       link-aggregation:
+         mode: balance-rr
+         slaves:
+         - eth1
+         - eth2
+----
++
+The foregoing exemplary YAML content is a policy manifest for bonding NICs `eth1` and `eth2` on the {product-title} cluster's nodes in a YAML file named `bond0-eth1-eth2_up.yaml`.
+
+. Apply the policy manifest.
++
+[source,bash]
+----
+[kni@node ~]$ oc apply -f bond0-eth1-eth2_up.yaml
```

Comment:
For each of the commands throughout this PR, do we need the preceding [kni@node ~] ?